.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -2,7 +2,7 @@
 name: "\U0001F4A1 Feature request"
 about: Suggest an idea for this project
 title: "[FEATURE REQUEST]"
-labels: user request
+labels: ''
```

Comment:
Why this change?

Reply:
Oh that might have been a mistake. I didn't mean to touch any of the other templates